docs/modules/eventbus.md

```diff
@@ -0,0 +1,26 @@
+# eventbus
+## **Overview** 
+Eventbus acts as an interface for sending/receiving messages on mqtt topics. It supports 3 kinds of mode: internalMqttMode,externalMqttMode and BothMqttMode.
```

Comment:
nit: `internalMqttMode`, `externalMqttMode` and `BothMqttMode` for a better look?

Reply:
better we can have a list to mention the modes : 
- internalMqttMode
- externalMqttMode
- bothMqttMode

Reply:
fixed